src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -72,7 +72,29 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
              'Once the property is enabled, the feature cannot be disabled. Currently only supported for LRS and '
              'ZRS replication types, hence account conversions to geo-redundant accounts would not be possible. '
              'For more information, please refer to https://go.microsoft.com/fwlink/?linkid=2086047.')
-
+    adds_type = CLIArgumentType(arg_type=get_three_state_flag(), min_api='2019-04-01',
+                                help='Enable Azure Files Active Directory Domain Service Authentication for '
+                                     'storage account. When --enable-files-adds is set to true, Azure Active '
+                                     'Directory Properties arguments must be provided.')
+    aadds_type = CLIArgumentType(arg_type=get_three_state_flag(), min_api='2018-11-01',
```

Comment:
should we add `arg_group = "Azure Active Directory Properties`? Just confirm

Reply:
For aadds, no azure active directory properties